charts/s3gw/templates/ingress-nginx.yaml

```diff
@@ -0,0 +1,75 @@
+{{- if .Values.ingress.nginx.enabled }}
```

Comment:
You can drop nginx manifest, since we discontinued it.
We are supporting only Traefik.